ios/FluentUI/Pill Button Bar/PillButton.swift

```diff
@@ -46,6 +53,7 @@ open class PillButton: UIButton, TokenizedControlInternal {
                                                name: PillButtonBarItem.titleValueDidChangeNotification,
                                                object: pillBarItem)
 
+        tokenSet.replaceAllOverrides(with: tokenOverrides)
```

Comment:
for "normal" cases, is it doing unnecessary iteration within replacealloverride because we will pass nil and it will try to remove overrides when there isn't anything that got override?

Reply:
Yeah, it would be. I'll add a nil check in the init.